app/api/schema/events.py

```diff
@@ -108,6 +108,8 @@ def validate_timezone(self, data, original_data):
     xcal_url = fields.Url(dump_only=True)
     average_rating = fields.Float(dump_only=True)
     order_expiry_time = fields.Integer(allow_none=True, default=10)
+    refund_policy = fields.String(allow_none=True,
+                                  default='All sales are final. No refunds shall be issued in any case.')
```

Comment:
Make it `dump_only` since we don't want to allow changing the refund policy. Once we support refunds, we will allow the event organizer to decide if he/she wants to support refunds or not.